lib/mpl_toolkits/mplot3d/proj3d.py

```diff
@@ -72,25 +72,33 @@ def rotation_about_vector(v, angle):
     return R
 
 
-def view_transformation(E, R, V, roll):
-    n = (E - R)
-    n = n/np.linalg.norm(n)
-    u = np.cross(V, n)
+def view_axes(E, R, V, roll):
+    """
+    Get the unit viewing axes in data coordinates.
+    `u` is towards the right of the screen
+    `v` is towards the top of the screen
+    `w` is out of the screen
```

Comment:
You renamed the arguments to the `view_transformation` function, but not `view_axes` here. It would be good to add numpydoc formatting here where you can as well since you're digging into these and it will help someone in the future for what these parameters should be.

Reply:
Wouldn't the easiest just be to create new functions here and leave the old ones untouched?

There are indeed quite a few more or less redundant function here anyway.

Reply:
Restored the original `view_transformation` function and moved this into a new `view_transformation_uvw` function.